flink-runtime/src/main/java/org/apache/flink/runtime/state/RegisteredBroadcastBackendStateMetaInfo.java

```diff
@@ -52,6 +52,23 @@ public RegisteredBroadcastBackendStateMetaInfo(
 		this.valueSerializer = Preconditions.checkNotNull(valueSerializer);
 	}
 
+	public RegisteredBroadcastBackendStateMetaInfo(RegisteredBroadcastBackendStateMetaInfo copy) {
+
+		Preconditions.checkNotNull(copy);
+
+		this.name = copy.name;
+		this.assignmentMode = copy.assignmentMode;
+		this.keySerializer = copy.keySerializer.duplicate();
+		this.valueSerializer = copy.valueSerializer.duplicate();
+	}
+
+	/**
+	 * Creates a deep copy of the itself.
+	 */
+	public RegisteredBroadcastBackendStateMetaInfo deepCopy() {
```

Comment:
Should not return raw type as well, but `RegisteredOperatorBackendStateMetaInfo<S>`.

Reply:
👍